eng/scripts/Language-Settings.ps1

```diff
@@ -15,7 +16,18 @@ function Get-javascript-PackageInfoFromRepo ($pkgPath, $serviceDirectory, $pkgNa
     $jsStylePkgName = $projectJson.name.Replace("@", "").Replace("/", "-")
     if ($pkgName -eq "$jsStylePkgName")
     {
-      return [PackageProps]::new($projectJson.name, $projectJson.version, $pkgPath, $serviceDirectory)
+      $pkgProp = [PackageProps]::new($projectJson.name, $projectJson.version, $pkgPath, $serviceDirectory)
+      if ($projectJson.name.StartsWith("@azure/arm"))
```

Comment:
We should be getting the sdk-type from https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-processor-host/package.json#L3. If this package is not a client package then we should update that property there.

Reply:
SDK_type property is not set for management and old SDK packages.